samples/templates/default-azuredeploy-docker-sql.json

```diff
@@ -150,6 +150,13 @@
             "metadata": {
                 "description": "Determine whether the sql schema should be automatically upgraded on server startup. If false, sql schema will not be initialized or upgraded on the server startup. The schema migration tool will be required to perform initialize or upgrade. If true, sql schema will be upgraded to the maximum supported version."
             }
+        },
+        "enableConvertData": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "Determines whether the $convert-data operation will be enabled for this fhir instance. If true, an Azure container registry will be created as part of the deployment. You will need owner or user-administrator permissions for this."
+            }
```

Comment:
FYI: The SQL deployment templates have been merged with the Cosmos DB templates (#1201) so these files don't exist anymore.

Reply:
That's awesome. We have merged the changes now.